spec/models/conference/importer_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
```

Comment:
Also, I added the content of the file to the bottom of this file, for your convenience in review. I'll remove them after review.